examples/experimental/imagenet_nas_arch_pytorch/model_def.py

```diff
@@ -11,25 +11,121 @@
 directories set up.
 """
 
-import os
 from collections import namedtuple
-from typing import Any, Dict, Tuple, cast
+from typing import Any, Dict
 
 import torch
+from torch.optim.lr_scheduler import _LRScheduler
 import torchvision.transforms as transforms
 from torch import nn
 
 import determined as det
 from data import ImageNetDataset
 from determined.pytorch import DataLoader, LRScheduler, PyTorchTrial, reset_parameters
 from model import NetworkImageNet
-from utils import AutoAugment, CrossEntropyLabelSmooth, Cutout, HSwish, Swish, accuracy
+from utils import (
+    RandAugment,
+    CrossEntropyLabelSmooth,
+    Cutout,
+    HSwish,
+    Swish,
+    accuracy,
+    AvgrageMeter,
+)
 
 Genotype = namedtuple("Genotype", "normal normal_concat reduce reduce_concat")
 
 activation_map = {"relu": nn.ReLU, "swish": Swish, "hswish": HSwish}
 
 
+class LinearLRScheduler(_LRScheduler):
```

Comment:
Question: Does this need a save_dict function? i.e. was this tested that the pause and reload functionally saves these values correctly?